lab/experiments/ThemeTest.vue

```diff
@@ -340,15 +353,30 @@ function contrastColors(bgHex) {
 	const isLight = chroma(bgHex).luminance() > IS_LIGHT_THRESHOLD;
 	const contrastColor = isLight ? '#000000' : '#ffffff';
 	const levels = isLight ? RATIOS.light : RATIOS.dark;
-	const colors = {};
+	const colors = {
+		'neutral-0': isLight ? '#ffffff' : '#000000',
+		'neutral-100': !isLight ? '#ffffff' : '#000000',
+	};
+
+	// if (isLight) {
+	// 	colors['neutral-90'] = 'rgb(142, 142, 147)';
+	// 	colors['neutral-80'] = 'rgb(174, 174, 178)';
+	// 	colors['neutral-20'] = 'rgb(229, 229, 234)';
+	// 	colors['neutral-10'] = 'rgb(242, 242, 247)';
+	// } else {
+	// 	colors['neutral-10'] = 'rgb(44, 44, 46)';
+	// 	colors['neutral-20'] = 'rgb(58, 58, 60)';
+	// 	colors['neutral-80'] = 'rgb(99, 99, 102)';
+	// 	colors['neutral-90'] = 'rgb(142, 142, 147)';
+	// }
 
 	Object.entries(levels).forEach(([name, level]) => {
-		colors[`color-${name}`] = chroma.mix(bgHex, contrastColor, level, 'lab').hex();
+		colors[`neutral-${name}`] = chroma.mix(bgHex, contrastColor, level, 'lab').hex();
 	});
 
 	return {
 		...colors,
-		'color-elevation': isLight ? '#ffffff' : colors['color-300'],
+		'color-elevation': isLight ? '#ffffff' : colors['neutral-30'],
```

Comment:
`colors['neutral-20']` for the dark backgrounds

Reply:
missed this one, thanks